src/tools/perf/libperf.c

```diff
@@ -1094,7 +1094,11 @@ static ucs_status_t uct_perf_setup(ucx_perf_context_t *perf, ucx_perf_params_t *
     uct_iface_params_t iface_params = {
         .tl_name     = params->uct.tl_name,
         .dev_name    = params->uct.dev_name,
+#if ENABLE_STATS
+        .stats_root  = ucs_stats_get_root(),
```

Comment:
ucs_stats_get_root(), returns type ucs_stats_node_t which can be used only when ENABLE_STATS defined.